jobs/build_hyperv.sh

```diff
@@ -3,11 +3,13 @@
 hyperv_node=$1
 JOB_TYPE=$2
 # Loading all the needed functions
-source /usr/local/src/cinder-ci/jobs/utils.sh
+source /usr/local/src/cinder-ci-2016/jobs/utils.sh
```

Comment:
Why not store the scripts base dir in a variable? We could have a config file here as well. It would be a lot easier to make such changes afterwards. A large amount of changes are required only because of this.